manifest/index.go

```diff
@@ -504,8 +505,16 @@ func (wixFile *WixManifest) Normalize() error {
 		if v.Major() > 255 || v.Minor() > 255 || v.Patch() > 65535 {
 			return fmt.Errorf("Failed to parse version '%v', fields must not exceed maximum values of 255.255.65535", wixFile.Version.User)
 		}
-		wixFile.Version.MSI = v.String()
-		wixFile.Version.Hex = v.Major()<<24 + v.Minor()<<16 + v.Patch()
+
+		if n, err := strconv.ParseInt(v.Metadata(), 10, 64); err == nil {
+			// append a metadata version number if present
+			wixFile.Version.MSI = fmt.Sprintf("%d.%d.%d.%d", v.Major(), v.Minor(), v.Patch(), n)
+			wixFile.Version.Hex = v.Major()<<32 + v.Minor()<<24 + v.Patch()<<8 + n
+		} else {
+			// only use major, minor, and patch if no numeric metadata
+			wixFile.Version.MSI = fmt.Sprintf("%d.%d.%d", v.Major(), v.Minor(), v.Patch())
+			wixFile.Version.Hex = v.Major()<<24 + v.Minor()<<16 + v.Patch()
```

Comment:
I think we want this to be something like this:
```suggestion
			wixFile.Version.Hex = v.Major()<<32 + v.Minor()<<24 + v.Patch()<<8 + 255
```

The idea being that v1.0.0 should come after v1.0.0-alpha+1, for instance